### PR TITLE
refactor(x/ecocredit): readable ecocredit query responses

### PR DESCRIPTION
**Pull request by ryanchristo**
## Description

Ref: #930

This pull request adds human-readable query responses for credit classes, projects, and credit batches.

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] followed the guidelines for [building modules](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules)
- [ ] included the necessary unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed `!` in the type prefix if API or client breaking change
- [ ] confirmed all author checklist items have been addressed 
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

**Comment by ryanchristo** [comment]
Should we do this for `CreditType` and `BatchBalance` as well to exclude the table row id? cc @aaronc @technicallyty

**Review by aaronc — COMMENTED** [review]
This makes me want to spend a bit of time on the query infrastructure in the SDK using Cosmos scalars so we don't have to manually write all this stuff. But generally looking good @ryanchristo

**Comment by aaronc** [comment]
> This makes me want to spend a bit of time on the query infrastructure in the SDK using Cosmos scalars so we don't have to manually write all this stuff. But generally looking good @ryanchristo

There are a few design questions though if I'm going to spend some time. Do we auto-generate gRPC endpoints that just convert bytes to strings for `cosmos.AddressBytes` scalars? Do we relate this to the value renderer work for sign mode textual? Or maybe we jump straight to graphql?

**Comment by ryanchristo** [comment]
> This makes me want to spend a bit of time on the query infrastructure in the SDK using Cosmos scalars so we don't have to manually write all this stuff. But generally looking good @ryanchristo

Thanks! Would be nice to not have to manually write this out.

> There are a few design questions though if I'm going to spend some time. Do we auto-generate gRPC endpoints that just convert bytes to strings for cosmos.AddressBytes scalars? Do we relate this to the value renderer work for sign mode textual? Or maybe we jump straight to graphql?

I think we should consider jumping straight to graphql given we need more than just addresses. Would be good to maybe continue this conversation in #930 given this is just the temporary solution.

File: x/ecocredit/server/core/query_classes_by_admin.go [hunk 1]
```diff
@@ -29,17 +29,21 @@ func (k Keeper) ClassesByAdmin(ctx context.Context, req *core.QueryClassesByAdmi
...
 		if err != nil {
 			return nil, err
 		}
-		var ci core.Class
-		if err = ormutil.PulsarToGogoSlow(v, &ci); err != nil {
-			return nil, err
+
+		info := core.ClassInfo{
+			Id:               class.Id,
+			Admin:            admin.String(),
```
Comment (technicallyty) [inline x/ecocredit/server/core/query_classes_by_admin.go:41]: we should call the `.String()` outside of the loop so it's only done once.

**Review by technicallyty — APPROVED** [review]
LGMT! - pending 1 nit & 1 perf fix

**Review by aleem1314 — APPROVED** [review]
Pre-approving

**Pull request merged**